### ISSUE TITLE
Pricing "start free trial" buttons go to different locations

### ISSUE THREAD

**Issue by joeduffy** [pulumi/docs#13118]
In this table, we have three "trial" buttons:

* Free - "Create an Organization"
* Team and Enterprise - "Start free trial"

![Image](https://github.com/user-attachments/assets/e05e8ff8-dc3d-4c6b-84a0-0be479fb1ffa)

I had expected these go to the same destination, but they are in fact pointing to different places:

* "Create an Organization" goes to https://app.pulumi.com/?create-organization=1
* "Start free trial" goes to https://app.pulumi.com/signup

The latter isn't really linking to a "free trial" page, it is a user signup page. Shouldn't they all point to the create-organization page?

**Comment by joeduffy** [comment]
I am still seeing that "Start new trial" still links to https://app.pulumi.com/signup/ when I am not signed in. We want it to point to the organization/trial, always, regardless of whether you are signed in. It's just the "Free" tier that should change between them (along with the text itself). So in summary, Team and Enterprise should be changed to unconditionally go to https://app.pulumi.com/signup/ and the rest is working as intended.

**Comment by shughes26** [comment]
Ah sorry, I misinterpreted the issue. Just to clarify - is the ask for the "Start Free Trial" buttons on Team and Enterprise to always point to the org creation link (https://app.pulumi.com/?create-organization=1) or to always point to the https://app.pulumi.com/signup/ signup link?

**Comment by arunkumar611** [comment]
@joeduffy  -  If a user is not signed in what do you expect https://app.pulumi.com/?create-organization=1?

Today if I'm not signed in, the Create Organization link takes me to the sign-in page. Do you expect users who click on "star trial" to already have a Pulumi account and they would just sign-in and create the org? My expectation is most of these users would be new to Pulumi, and it makes sense to get them to the sign-up page.

**Comment by joeduffy** [comment]
> Just to clarify - is the ask for the "Start Free Trial" buttons on Team and Enterprise to always point to the org creation link (https://app.pulumi.com/?create-organization=1)

Correct. The start trial button should always take you to the start trial page.

> Today if I'm not signed in, the Create Organization link takes me to the sign-in page. Do you expect users who click on "star trial" to already have a Pulumi account and they would just sign-in and create the org? My expectation is most of these users would be new to Pulumi, and it makes sense to get them to the sign-up page.


Doesn't going to https://app.pulumi.com/?create-organization=1 automatically take you through a sign up flow if you aren't already signed up?

**Comment by arunkumar611** [comment]
If you are not signed in, Create Organization takes you to the sign-in page. 

![Image](https://github.com/user-attachments/assets/d01063de-3d3e-4234-aa29-b07bedce60a5)

If you click sign-up when we maintain the "create organization" in the URL - we ignore it in the onboarding flow. 

![Image](https://github.com/user-attachments/assets/efdea8d7-2f33-418e-a669-bbf3a25fb99b)

![Image](https://github.com/user-attachments/assets/9e993c0d-b147-4027-99a2-2212d8775565)

![Image](https://github.com/user-attachments/assets/0db1ad04-9cc0-4c88-9723-9fb3ab7bafb7)

**Comment by arunkumar611** [comment]
@joeduffy  - I think your intent is when "users are not signed-in, clicking on start-trial, takes you to the sign-up flow and automatically asks you to create an Org, giving you to two weeks free trial"- Is that correct?

**Comment by joeduffy** [comment]
Why do we show you the "I signed up for Pulumi Cloud because" selector, when the presence of `?create-organization=1` already answers that for us: "I want to set up Pulumi for my team." Isn't this a Pulumi Cloud bug?

**Comment by arunkumar611** [comment]
Yes, indeed.

We need to handle this in Pulumi Cloud. I'll create an issue for it. Once we fix the flow in Pulumi Cloud, we can update the pricing page. 

For now, I recommended the "start trial" button to point to the sign-up URL until we fix it.

**Comment by joeduffy** [comment]
> For now, I recommended the "start trial" button to point to the sign-up URL until we fix it.

Why? It accomplished the same thing - you signed up - and if we fix the docs, then once we fix the app, everything will be as it should be. (And note, even with that fix, you'll land on `/signin` by default anyway, so basically the same experience minus one click...)

**Comment by arunkumar611** [comment]
Here is the issue: https://github.com/pulumi/pulumi-service/issues/25208

> Why? It accomplished the same thing - you signed up - and if we fix the docs, then once we fix the app, everything will be as it should be. (And note, even with that fix, you'll land on /signin by default anyway, so basically the same experience minus one click...)
Here is the ideal flow in my mind: 
- If the user is signed in, and clicks on start trial, we take them to the "Create an Organization" dialogue within the app (This works today)
- If the user is **not** signed in and clicks on start trial, we take them to the sign-up page. Once they enter username, password, and email, we ask them for the Organization name (i.e. show this screen)

![Image](https://github.com/user-attachments/assets/68ebbb24-fe09-408a-85a4-a2a5073afdb5)

One less click. Additionally, the "create an account" isn't so visible to users. This flow is optimized for New Users to start a trial.
